### PR TITLE
start the release notes

### LINKED ISSUE #71: Add release history per release process
https://nsls-ii.github.io/scientific-python-cookiecutter/publishing-releases.html

### LINKED ISSUE #72: Organize source documentation into API Reference section
Compare and similar with https://blueskyproject.io/ophyd/

Needs content for two new pages:

- [ ] overview (issue #122)
- [x] ready_to_use (issue #123)

### LINKED ISSUE #74: no version number available from import!!!
```
In [1]: import gi
   ...: gi.require_version("Hkl", "5.0")

In [2]: import hkl

In [3]: hkl.__version__
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-25eb45a6aff7> in <module>
----> 1 hkl.__version__

AttributeError: module 'hkl' has no attribute '__version__'

```

yet

```
(base) ~/.../Bluesky/hklpy $ pwd
~/Documents/projects/Bluesky/hklpy
(base) ~/.../Bluesky/hklpy $ python ./setup.py version
running version
keywords are unexpanded, not using
got version from VCS {'version': '0.3.14+250.gb2652eb', 'full-revisionid': 'b2652eb57e4ccc632ad2a1891bb5bf2c641810aa', 'dirty': False, 'error': None}
Version: 0.3.14+250.gb2652eb
 full-revisionid: b2652eb57e4ccc632ad2a1891bb5bf2c641810aa
 dirty: False
```

### PR DESCRIPTION
**Pull request by prjemian**
fixes #71, fixes #74

> History starts today.

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Diffractometer wavelength **must** use *angstrom* units to match the
+  lattice constants.  Previously, wavelength was stated to be in
+  `nm`. Instruments upgrading to this release should verify the units
+  actually in use.
+
+New Features and/or Enhancements
+--------------------------------
+
+* Diffractometer energy units can be specified.  Unit conversions
+  are handled by the *pint* [#]_ package.
```
Comment (mrakitin) [inline docs/source/release_notes.rst:36]: Did we ever include `pint` as a dependency?

**Review by mrakitin — COMMENTED** [review]
Looks good. One question about dependencies.

**Comment by prjemian** [comment]
Good point.  I'll check.  Others as well, for the notebooks.

**Comment by mrakitin** [comment]
Sounds good to add it too. We already have a heavy depensency of `hkl`, so I don't see why not to add more, if they are used.

File: docs/source/release_notes.rst (hunk 1 above)
Comment (prjemian) [reply docs/source/release_notes.rst:36]: yes

**Review by mrakitin — APPROVED** [review]
Looks good. Had a comment about an extra conda channel, but we can discuss it later.

**Comment by prjemian** [comment]
Thanks, @mrakitin !

**Pull request merged**